src/test/java/com/github/mauricioaniche/ck/TightClassCohesionTest.java

```diff
@@ -0,0 +1,54 @@
+package com.github.mauricioaniche.ck;
+
+import junit.framework.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+public class TightClassCohesionTest extends BaseTest {
+    private static Map<String, CKClassResult> report;
+
+    @BeforeAll
+    public static void setUp() {
+        report = run(fixturesDir() + "/ClassCohesion");
+    }
+
+    @Test
+    public void highCohesion() {
+        CKClassResult ckClass = report.get("ClassCohesion.HighCohesion");
+        Assert.assertEquals(1f, ckClass.getTightClassCohesion());
+    }
+
+    @Test
+    public void noCohesion() {
+        CKClassResult ckClass = report.get("ClassCohesion.NoCohesion");
+        System.out.println(ckClass.getVisibleMethods());
+        Assert.assertEquals(0f, ckClass.getTightClassCohesion());
+    }
+
+    @Test
+    public void mediumCohesion1() {
+        CKClassResult ckClass = report.get("ClassCohesion.MediumCohesion");
+        Assert.assertEquals(2f / 6f, ckClass.getTightClassCohesion());
+    }
+
+    @Test
+    public void mediumCohesion2() {
+        CKClassResult ckClass = report.get("ClassCohesion.Simple1");
+        Assert.assertEquals(2f / 10f, ckClass.getTightClassCohesion());
+    }
+
+    @Test
+    public void mediumCohesion3() {
+        CKClassResult ckClass = report.get("ClassCohesion.Simple2");
+        Assert.assertEquals(4f / 10f, ckClass.getTightClassCohesion());
```

Comment:
in float assertions, it's good to add a _delta_ at the end of the assertion, to make sure the test doesn't break.

`Assert.assertEquals(a, b, 0.00001)` for example

Reply:
Thanks for the advice. I added it.